assets/src/edit-story/components/panels/design/textStyle/textStyle.js

```diff
@@ -30,16 +30,14 @@ import StyleControls from './style';
 import ColorControls from './color';
 import FontControls from './font';
 
-function StylePanel(props) {
-  const fontDropdownRef = useRef(null);
-  const textColorRef = useRef(null);
+const StylePanel = forwardRef(function StylePanel(props, ref) {
   // use highlights to update panel styles
   // but don't dynamically adjust the `isPersistable` prop on `SimplePanel`
   // the textStyle panel automatically opens already whenever a text element is selected
   // if we update this to only be when there's a highlight the functionality that is expected
   // will be wrong.
-  const dropdownHighlight = useFocusHighlight(states.FONT, fontDropdownRef);
-  const colorHighlight = useFocusHighlight(states.TEXT_COLOR, textColorRef);
+  const dropdownHighlight = useFocusHighlight(states.FONT, ref);
+  const colorHighlight = useFocusHighlight(states.TEXT_COLOR, ref);
```

Comment:
We need these separate refs to highlight the given dropdown. 

![text panel highlights](https://user-images.githubusercontent.com/10720454/123827514-89f2c100-d8b5-11eb-8f2e-f88a4cc56701.gif)